go.mod

```diff
@@ -3,15 +3,16 @@ module github.com/makerdao/vdb-mcd-transformers
 go 1.15
 
 require (
-	github.com/ethereum/go-ethereum v1.9.21
+	github.com/ethereum/go-ethereum v1.9.22
 	github.com/jmoiron/sqlx v1.2.0
 	github.com/lib/pq v1.0.0
+	github.com/makerdao/vdb-transformer-utilities v0.0.0-20201015220351-4d84cb49b1ad
 	github.com/makerdao/vulcanizedb v0.1.0
-	github.com/onsi/ginkgo v1.14.0
-	github.com/onsi/gomega v1.10.1
-	github.com/sirupsen/logrus v1.2.0
+	github.com/onsi/ginkgo v1.14.1
+	github.com/onsi/gomega v1.10.2
+	github.com/sirupsen/logrus v1.7.0
 	github.com/spf13/cobra v0.0.3
-	github.com/spf13/viper v1.3.2
+	github.com/spf13/viper v1.7.1
```

Comment:
A little nervous about these deps updating, but also wondering if we might want to go ahead and update them on the vulcanizedb side since this happened automatically

Reply:
Your nerves were right - you probably saw that this failed to start the execute container.